debian/postinst

```diff
@@ -26,13 +26,27 @@ SYSTEMD_HELPER_ENABLED_AUTO_ATTACH_DSH="/var/lib/systemd/deb-systemd-helper-enab
 SYSTEMD_HELPER_ENABLED_WANTS_LINK="/var/lib/systemd/deb-systemd-helper-enabled/multi-user.target.wants/ua-auto-attach.service"
 
 
+check_esm_infra_is_installed() {
+    expected_output="enabled"
+    actual_output=$(ua status | grep esm-infra | awk '{print $3}')
+
+    if [ "$expected_output" = "$actual_output" ]; then
+        return 0
+    else
+        return 1
+    fi
+}
+
```

Comment:
Sorry consolidating two review comments into one.
1. rename
2. What do you think about this (in case we change column ordering in the TUI at some point we don't want to print $3 to fail)

```suggestion
check_esm_infra_is_enabled() {
    return $(ua status | egrep -q 'esm-infra.*enabled')
}

```